docs/registry.md

```diff
@@ -57,7 +57,7 @@ Registries that are not listed have not been tested by the SOCI maintainers or r
 | ----------------------------------------------------------------------------------------- | ----------- | ------------- | ---------------------------------------------------- |
 | [Docker Hub](https://hub.docker.com)                                                      | No          | N/A           | Does not support image manifests with subject fields |
 | [Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/ecr/)                    | Yes         | Fallback      |                                                      |
-| [Amazon ECR Public Gallery](https://gallery.ecr.aws)                                      | No          | N/A           | Does not support image manifests with subject fields |
+| [Amazon ECR Public Gallery](https://gallery.ecr.aws)                                      | Yes          | N/A           | Does not support image manifests with subject fields |
```

Comment:
nit: can you also replace `N/A` with `Fallback` and remove the comment about not supporting manifests with subject fields?

Reply:
Yup! Done